build/verify-test-urls.sh

```diff
@@ -1,70 +0,0 @@
-#! /bin/bash
-
-# Automatically exit on error
-set -e
-
-ANNOTATIONS=$(mktemp)
-ERRORS=0
-
-# Keep short because GitHub doesn't wrap it
-ERROR_MESSAGE="Every new or edited feature must be tested manually before merging.
-To help testing, we're progressively adding test URLs in each feature.
-
-You can find or create a test URL:
-
-- on our sandbox repo: https://github.com/refined-github/sandbox
-- on previous PRs for this feature
-
-The section must be appended to each .tsx file:
-
-/*
-
-Test URLs:
-
-https://github.com/a/REAL/url/here
-https://github.com/another/REAL/url/here
-
-*/
-"
-
-# Replace line breaks with "%0A"
-# https://github.com/actions/toolkit/issues/193#issuecomment-605394935
-ERROR_MESSAGE=${ERROR_MESSAGE//$'\n'/%0A}
-
-echo WILL VERIFY:
-echo
-
-for FILE in "$@"; do
-	# skip github-bugs.css
-	if [ "$FILE" = "source/features/github-bugs.css" ]; then
-		continue
-	fi
-
-	IS_FILE_CSS=$(echo "$FILE" | grep -q "\.css$" && echo 1 || echo 0)
-	SIBLING_TSX_FILE=$(echo "$FILE" | sed 's/\.css$/.tsx/')
-	DOES_SIBLING_EXIST=$(test -f "$SIBLING_TSX_FILE" && echo 1 || echo 0)
-	if test "$IS_FILE_CSS" -eq 1 && test "$DOES_SIBLING_EXIST" -eq 1; then
-		continue
-	fi
-
-	LAST_LINE=$(wc -l < "$FILE")
-	if grep -q "test url" -i "$FILE"; then
-		echo ✅ "$FILE"
-		echo "::notice file=$FILE,line=$LAST_LINE::✅" >> "$ANNOTATIONS"
```

Comment:
It would be nice to still output [this positive annotation](https://github.com/refined-github/refined-github/pull/6680#discussion_r1199764922) on all files though, just to bring the test URLs to light when reviewing, but it seems it's already not working?